gptables/test/test_wrappers.py

```diff
@@ -547,3 +546,57 @@ def test_make_table_of_contents(self, testbook, create_gptable_with_kwargs,
         exp_contentsheet.table = None
 
         assert got_contentsheet.__dict__ == exp_contentsheet.__dict__
+
+    def test_make_notesheet(self, testbook):
+        """
+        Test that creating a notes table sheet using `make_notesheet` generates
+        the same gptables.GPTable object as expected.
+        """
+        gp_workbook = testbook.wb
+        gp_workbook._annotations = [1, 2]
+        dummy_table = pd.DataFrame(data={"Note number":[1, 2], "Note text":[3, 4]})
+        
+        notes_name = "Just_a_notesheet"
+        notes_title = "Are these the notes you're looking for?"
+        notes_instructions = "These are not the notes you're looking for"
+        
+        got_notesheet = gp_workbook.make_notesheet(notes_table=dummy_table,
+                                                   table_name=notes_name,
+                                                   title=notes_title,
+                                                   instructions=notes_instructions)
+        exp_notesheet = gptables.GPTable(table=dummy_table,
```

Comment:
For the gptables tests, we have been using the fixture `create_gptable_from_kwargs` so that we can create a GPTable with only the arguments we care about in that test. It would be good to do the same here to keep it consistent